src/main/webapp/app/entities/result/result.component.ts

```diff
@@ -1,13 +1,24 @@
 import { Component, Input, OnChanges, OnInit, SimpleChanges } from '@angular/core';
-import { ParticipationService, StudentParticipation } from 'app/entities/participation';
+import { ParticipationService, StudentParticipation, InitializationState } from 'app/entities/participation';
 import { Result, ResultDetailComponent, ResultService } from '.';
 import { RepositoryService } from 'app/entities/repository/repository.service';
 import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
 import { HttpClient } from '@angular/common/http';
+import { Course } from 'app/entities/course';
 import { ExerciseType } from 'app/entities/exercise';
 import { MIN_POINTS_GREEN, MIN_POINTS_ORANGE } from 'app/app.constants';
 import { TranslateService } from '@ngx-translate/core';
 import { JhiWebsocketService } from 'app/core';
+import * as moment from 'moment';
+
+const enum ResultTemplateStatus {
+    IS_BUILDING = 'is-building',
```

Comment:
I would normally assign the same string as the enum qualifier:

`IS_BUILDING = 'IS_BUILDING'`

When debugging this is clearer.